docs/API/DI/ServiceNotFoundException.md

```
@@ -0,0 +1,91 @@
+## Source
+
+[View source code](https://github.com/evansims/openfga-php/blob/main/src/DI/ServiceNotFoundException.php)
+
+## Implements
+
+* `Throwable`
+* `Stringable`
+
+## Methods
+
+#### getCode
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)